.github/ISSUE_TEMPLATE/new-hub.yml

```
@@ -0,0 +1,164 @@
+        Answer the sections below as best you can.
+        We can update information as we learn more.
+
+        ## Community Information
+
+  - type: textarea
+    id: background
+    attributes:
+      label: Hub Description
+      description: |
+        Two or three sentences about the community and use-case for the hub.
+    validations:
+      required: true
+
+  - type: input
+    id: community-representative
+    attributes:
+      label: Community Representative
+      description: |
+        The main point-of-contact for this hub, and the person who requests changes to the hub on behalf of the community.
+        **This person will be the first Hub Administrator**, and may add new hub administrators from the JupyterHub UI.
+        Should be either a GitHub handle or email address, depending on the type of authentication you've indicated above.
+    validations:
+      required: true
+
+  - type: input
+    id: target-start-date
+    attributes:
+      label: Target start date
+      description: |
+        The target start date when this hub will begin operation.
+        This should be at least a week before any important usage of the hub will occur, to ensure that the Hub Representative has time to try it out. 
+    validations:
+      required: true
+
+  - type: input
+    id: important-dates
+    attributes:
+      label: Important dates
+      description: |
+        Any important dates that we should consider for this hub.
+        For example, if it will be used for a class, the starting day of class and any days with tests.
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Hub Customization
+
+  - type: input
+    id: auth-type
+    attributes:
+      label: Hub Authentication Type
+      description: |
+        What kind of authentication service will be used for this community.
+        This boils down to what kind of "log-in username" users wish to use on the hub.
+        Currently, the two options are **GitHub handles** (e.g., `@mygithubhandle`) or Google OAuth usernames (e.g., `myname@2i2c.org`).
+        You must choose **only one option**.
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: features
+    attributes:
+      label: Extra features you'd like to enable
+      description: |
+        In addition to a base JupyterHub, you may enable extra infrastructure features that provide additional functionality.
+        These generally come with added complexity and cost, so make sure you've discussed these options with somebody at 2i2c before selecting them here.
+
+      options:
+        - label: Dedicated Scalable Dask Cluster with [Dask Gateway](https://gateway.dask.org/)
+    validations:
+      required: false
+
+  - type: input
+    id: hub-logo
+    attributes:
+      label: Hub logo
+      description: |
+        A URL to an image that will be used on the hub's landing page. (optional)
+    validations:
+      required: true
```

```suggestion
      required: false
```
You've stated in the description that this is optional, but the issue form is expecting it to be required 😄